app/assets/javascripts/js-processor.js

```diff
@@ -119,6 +133,19 @@ globalThis.transpile = function (source, options = {}) {
   const { moduleId, filename, skipModule, themeId, commonPlugins } = options;
   const plugins = [];
 
+  // if (GJS) {
```

Comment:
Yeah I think we should make this conditional on the file extension - we don't wanna confuse things by accepting `<template>` in regular JS files